### DISCUSSION TITLE
Ubuntu instead of xubuntu?

### DISCUSSION THREAD

**Discussion by JetpackJackson** [dnschneid/crouton#4565] [Q&A] [answered]
I used crouton to install the example Ubuntu install with xfce. Worked great, perfect. I was wondering what I have to change in the install command to install "regular" Ubuntu instead to be able to switch between chrome os and Ubuntu. Thanks in advance and thank you for crouton!

**Comment by DennisLfromGA** [comment, marked as answer]
@JetpackJackson,

If your device [supports it](https://github.com/reynhout/chrx#chromebooks) , you can use `chrx` to install a Linux distro and dual-boot it. See the link below for details -   
- reynhout/chrx: Chromebook Unix
https://github.com/reynhout/chrx#chromebooks

*Hope this helps,*
-**DennisLfromGA**

**Reply by JetpackJackson** [reply]
Thank you for replying! Sadly my chromebook has an ARM processor so it does not appear to be supported for chrx usage. Thank you though!

**Comment by DennisLfromGA** [comment]
@JetpackJackson,

No problem, sorry to hear it's not feasible for your device. 😔

-**DennisLfromGA**
